documentdb/spring-data-azure-documentdb/src/main/java/com/microsoft/azure/spring/data/documentdb/core/DocumentDbTemplate.java

```diff
@@ -311,4 +353,24 @@ private String getCollectionLink(String databaseName, String collectionName) {
     private String getDocumentLink(String databaseName, String collectionName, String documentId) {
         return getCollectionLink(databaseName, collectionName) + "/docs/" + documentId;
     }
+
+    private String getPartitionKeyPath(String partitionKey) {
+        return "/" + partitionKey;
+    }
+
+    private RequestOptions getRequestOptions(String partitionKeyValue, Integer requestUnit) {
```

Comment:
I see all the caller of this function one of partitionKeyValue and requestUnit is null? That means this function always return null even for the partition case?

Reply:
when collection is not partitioned, partitionkey=null, this method will return null reuqestoption
when collection is partitioned, partitionkey has value, this method will return requestoption object.
callers of this method is in DocumentDBTemplate.java, methods in template just pass in partitionkey from repository level. if user choose partition, pass partitionkey in, then requestoption will set partitionkey. null is requestUnit, which only need specify when create collection. when operate with document, no need this param, so null. get this logic from documentdb sdk test